app/src/pages/Player/components/AchievementGroup/AchievementGroup.jsx

```diff
@@ -92,6 +93,10 @@ function formatThreshold(threshold) {
     return getLevel(threshold + 100).toString();
   }
 
+  if ([273742 * TOTAL_SKILLS, 737627 * TOTAL_SKILLS, 1986068 * TOTAL_SKILLS, 5346332 * TOTAL_SKILLS, 13034431 * TOTAL_SKILLS].includes(threshold)) {
```

Comment:
`[273742 * TOTAL_SKILLS, 737627 * TOTAL_SKILLS, 1986068 * TOTAL_SKILLS, 5346332 * TOTAL_SKILLS, 13034431 * TOTAL_SKILLS]`

could probably be rewritten as

`[273742, 737627, 1986068, 5346332, 13034431].map(i => i * TOTAL_SKILLS)`

to look a little cleaner

Reply:
And if that's the case, should probably remove the `TOTAL_SKILLS` and `REAL_SKILLS` as they're now only used once, and instead write that map as 

`[273742, 737627, 1986068, 5346332, 13034431].map(i => i * SKILLS.filter(s => s !== 'overall').length)`